cumulusci/core/keychain/base_project_keychain.py

```diff
@@ -160,6 +159,9 @@ def list_orgs(self):
     def _raise_org_not_found(self, name):
         raise OrgNotFound(f"Org named {name} was not found in keychain")
 
+    def cleanup_org_cache_dirs(self):
+        raise NotImplementedError("Subclasses must implement this method.")
```

Comment:
Let's make this pass rather than raising. In other words, make it so there is a default implementation which does nothing, rather than requiring subclasses to implement it as part of implementing the keychain contract. That way if there is a subclass of BaseProjectKeychain outside of cumulusci (I think there is in MetaCI), we won't break it.